patches/node/fix_lazyload_fs_in_esm_loaders_to_apply_asar_patches.patch

```diff
@@ -60,28 +60,19 @@ index acfcc1f1f92548117ded857fa8d4dd140e07ab2b..ce8092b96aee8d09ff382110db4be62d
      });
      const { search, hash } = resolved;
 diff --git a/lib/internal/modules/esm/translators.js b/lib/internal/modules/esm/translators.js
-index 178f362b2869173d51b2e97a8d35a16ab4ed4962..4c3a0d8c484a402fe419a0bd45c7e2b1d717cb4a 100644
+index bda102b266b17936efc453d039213517c4ef921c..dbbe37474c70addfbf8cf805c29732dcd38c74f6 100644
 --- a/lib/internal/modules/esm/translators.js
 +++ b/lib/internal/modules/esm/translators.js
-@@ -25,7 +25,7 @@ function lazyTypes() {
- }
+@@ -32,7 +32,7 @@ function lazyTypes() {
  
+ const { containsModuleSyntax } = internalBinding('contextify');
  const assert = require('internal/assert');
 -const { readFileSync } = require('fs');
 +const fs = require('fs');
  const { dirname, extname, isAbsolute } = require('path');
  const {
-   hasEsmSyntax,
-@@ -132,7 +132,7 @@ translators.set('module', async function moduleStrategy(url, source, isMain) {
-  */
- function enrichCJSError(err, content, filename) {
-   if (err != null && ObjectGetPrototypeOf(err) === SyntaxErrorPrototype &&
--      hasEsmSyntax(content || readFileSync(filename, 'utf-8'))) {
-+      hasEsmSyntax(content || fs.readFileSync(filename, 'utf-8'))) {
```

Comment:
I think this block of code that we were patching has been moved to C++ -- does that new codepath still pick up our patched fs?

Asking not as a "change requested" but because I don't think I understand the implications of this line going away